thirdparty/faiss/faiss/IndexAdditiveQuantizer.cpp

```diff
@@ -93,6 +93,7 @@ struct AQDistanceComputerLUT : FlatCodesDistanceComputer {
             bias = 0;
         } else {
             bias = fvec_norm_L2sqr(x, d);
+            bias = (bias == 0.0 ? 1.0 : bias);
```

Comment:
no, because it uses norm for distance = x^2 - 2xy + y^2 calculation, not for a cosine-related code